src/Components/GEOSldas_GridComp/GEOSmetforce_GridComp/LDAS_Forcing.F90

```diff
@@ -287,130 +270,60 @@ subroutine get_forcing( date_time, force_dtstep, met_path, met_tag,        &
-       
-       call repair_forcing( N_catd, met_force_obs_tile_new, &
-            echo=.true., tile_coord=tile_coord )
-       
     elseif (index(met_tag, 'Princeton_netcdf')/=0) then ! tyamada+reichle, 17 Jul 2007   
        
        call get_Princeton_netcdf(  date_time_tmp, met_path, N_catd, tile_coord, &
             met_force_obs_tile_new, nodata_forcing)
        
-       ! check for nodata values and unphysical values
-       ! (check here, not outside "if" block, because of GEOSgcm case)
-       
-       call check_forcing_nodata( N_catd, tile_coord, nodata_forcing, &
-            met_force_obs_tile_new )
-       
-       call repair_forcing( N_catd, met_force_obs_tile_new, &
-            echo=.true., tile_coord=tile_coord )
-       
     elseif (index(met_tag, 'conus_0.5d_netcdf')/=0) then ! sarith+reichle, 17 Jul 2007   
        
        call get_conus_netcdf(  date_time_tmp, met_path, N_catd, tile_coord, &
             met_force_obs_tile_new, nodata_forcing)
        
-       ! check for nodata values and unphysical values
-       ! (check here, not outside "if" block, because of GEOSgcm case)
-       
-       call check_forcing_nodata( N_catd, tile_coord, nodata_forcing, &
-            met_force_obs_tile_new )
-       
-       call repair_forcing( N_catd, met_force_obs_tile_new, &
-            echo=.true., tile_coord=tile_coord )
-       
     else ! assume forcing from GEOS5 GCM ("DAS" or "MERRA") output
        
        if(root_logit) write (logunit,*) 'get_forcing(): assuming GEOS-5 forcing data set'
        
        ! subroutine get_GEOS() provides backward-looking fluxes 
        
        bkwd_looking_fluxes            = .true.
-
-       call get_GEOS( date_time_tmp, force_dtstep, met_path, met_tag,       &
-            N_catd, tile_coord, MET_HINTERP, AEROSOL_DEPOSITION,            &
-            supported_option_MET_HINTERP,                                   &
-            supported_option_AEROSOL_DEPOSITION,                            &            
-            met_force_obs_tile_new, nodata_forcing, MERRA_file_specs,       &
-            init )
        
-       ! check for nodata values and unphysical values
-       ! (check here, not outside "if" block, because of GEOSgcm case)
-       
-       call check_forcing_nodata( N_catd, tile_coord, nodata_forcing, &
-            met_force_obs_tile_new )    
-
-       ! call repair_forcing with switch "unlimited_Qair=.true." for GEOS5 forcing
+       ! call repair_forcing with switch "unlimited_Qair=.true." 
        ! (default is to limit Qair so that it does not exceed Qair_sat)
        ! reichle+qliu,  8 Oct 2008
        ! 
        ! likewise for "unlimited_LWdown=.true."
        ! reichle, 11 Feb 2009       
        
-       call repair_forcing( N_catd, met_force_obs_tile_new, &
-            echo=.true., tile_coord=tile_coord,             &
-            unlimited_Qair=.true., unlimited_LWdown=.true. )
+       unlimited_Qair                 = .true.
```

Comment:
move the two lines after get_GEOS so it is consistent with ge_ERA5_LIS and shows they are not used in the calls

Reply:
Thanks! I rearranged the statements (and a couple more things) for clarity.